tests/tools.vitruv.applications.umljava.tests/src/tools/vitruv/applications/umljava/tests/uml2java/UmlToJavaAttributeTest.xtend

```diff
@@ -231,6 +232,56 @@ class UmlToJavaAttributeTest extends AbstractUmlToJavaTest {
 		override setupTransformationDirection() {
 			configureBidirectionalExecution()
 		}
+
+	   @Test
+        override testCreatePrimitiveAttribute() {
+            super.testCreatePrimitiveAttribute()
+            validateUmlView [
+                val umlClass = defaultUmlModel.claimClass(CLASS_NAME)
+                assertUmlClassDontHaveOperation(umlClass, buildGetterName(ATTRIBUTE_NAME))
```

Comment:
Not part of your PR, but we should probably rename this method to `assertUmlClassHasNoOperation`, as it only takes only class and thus "has" fits better than "have"! 😄